keras_rcnn/backend/tensorflow_backend.py

```diff
@@ -140,7 +140,7 @@ def crop_and_resize(image, boxes, size):
     box_ind = box_ind[:, 0]
     box_ind = keras.backend.reshape(box_ind, [-1])
 
-    boxes = keras.backend.reshape(boxes, [-1, 4])
+    boxes = keras.backend.cast(keras.backend.reshape(boxes, [-1, 4]),'float32')
```

Comment:
Can you change it to using keras.backend.floatx() rather than 'float32' or 'float64'?

Reply:
As I described in the commit message, Tensorflow does not automatically cast types for `CropAndResize` Op so in some cases that numpy generates float64 arrays, not casting will result in the following error:

```
TypeError: Input 'boxes' of 'CropAndResize' Op has type float64 that
does not match expected type of float32.
```

To prevent it, we should make sure boxes are casted.